decidim-admin/config/locales/en.yml

```diff
@@ -825,9 +826,15 @@ en:
           update: Update
         form:
           colors:
+            choose_color: Choose a primary color
             colors_title: Organization colors
             colors_warning_html: Warning! Changing these colors can break the accessibility contrasts. You can check the contrast of your choosing with <a href="%{link}">WebAIM Contrast Checker</a> or other similar tools.
+            explanation: This tool helps you out to choose a color scheme, made up of hues equally spaced around the color wheel, that will be used in the organization's website.
             header_snippets_help: Use this field to add things to the HTML head. The most common use is to integrate third-party services that require some extra JavaScript or CSS. Also, you can use it to add extra meta tags to the HTML. Note that this will only be rendered in public pages, not in the admin section.
+            legend: Main application colors, based on the <a href="%{link}">Triadic algorithm</a>. The <i>secondary</i> color is auto-calculated from the primary color and the saturation you chose.
```

Comment:
@alecslupu this is the only thing about your review that I didn't take into account. 

> [Additionally, could you mark this link as external ?](https://github.com/decidim/decidim/pull/9419#discussion_r1070914820)
 
 I preferred to use the same pattern as [the color_warning_html key](https://github.com/decidim/decidim/blob/016d1f470570827fea51bc2b92209cdab683f328/decidim-admin/config/locales/en.yml#LL831C1-L831C222), on line 831, as it should behave the same IMHO. Also, as far as I know, the external link indicator (aka icon) isn't enabled in the admin panel (if that's what you meant).

Either case if we change this we should review the rest of the external links in the admin to be consistent (and that should be handled outside of this PR too)